lib/util/author.py

```diff
@@ -3,9 +3,8 @@
 from lib.util.read import lines_per_n
 
 
-def get_uid_map(write_to_file=True):
+def get_uid_map(input_file='clean_data.json',output_file="author_uid_map.json"):
     """
-
     This function is used to generate and write to a JSON file the mapping of authors to a unique integer identifier.
     Authors are identified through a regular expression search for their email addresses. The integer identifiers
     generated are used in other modules like the generation and statistical analysis of hyperedges.

```

Comment:
@tejalwakchoure please update the function parameters in the sphinx docs. For an example, see [here](https://github.com/DeveloperCAP/MLCAT/blob/development/lib/analysis/author/community.py#L25).